core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/AbstractLibrary.java

```diff
@@ -131,7 +131,7 @@
   @JoinColumn(name = "lastModifier", nullable = false)
   private User lastModifier;
 
-  @Formula(value = "(SELECT MAX(lcl.changeTime) FROM LibraryChangeLog lcl WHERE lcl.libraryId = libraryId)")
+  @Formula(value = "(SELECT MAX(lmod.changeTime) FROM LibraryChangeLog lmod WHERE lmod.libraryId = libraryId)")
```

Comment:
(Also, the Hibernate-no-longer-loading-Libraries change was merged after I opened the PR ;) but I'll update this to remove the annotation.)